### PR TITLE
Miscellaneous tidy up

### PR DESCRIPTION
**Pull request by LukeShu**
This is a bunch of minor tidying up, from various WIP feature branches, that I figured I could go ahead and try to land.

 - Tighten up linter settings around imports
 - `pkt/dlog.WrapTB`: Add a flag to have `.Errorln()` not cause a test failure
 - `pkg/supervisor`: Split into multiple files; give each datatype its own file.  Also fix the re-awakened linter complaints caused by that. XXX
 - `Makefile`: Set `.DEFAULT_GOAL = help`
 - `mv g.md docs/watt-old.md`
 - ~~Use `os.TempDir()` instead of hard-coding `"/tmp"`~~
 - Rename `logexec` to `dexec` (per previous discussion)

The one tagged "XXX" is the only one that I might expect anyone to take issue with.

### CODE REVIEW

File: cmd/edgectl/main.go [hunk 1]
```diff
@@ -15,7 +16,9 @@ import (
 var Version = "(unknown version)"
 
 const socketName = "/var/run/edgectl.socket"
-const logfile = "/tmp/edgectl.log"
+
+var logfile = filepath.Join(os.TempDir(), "edgectl.log")
```
Comment (ark3) [inline cmd/edgectl/main.go]: What does this yield on a Mac? This needs to be a user-friendly path.

Comment (ark3) [reply cmd/edgectl/main.go]: BTW, this is _also_ a machine-scoped file. Is this change really appropriate?

Comment (LukeShu) [reply cmd/edgectl/main.go]: I guess not.  Removing that bit from the commit.

**Comment by LukeShu** [comment]
Well, dropping the /tmp patch seems to have made the flakes go away :shrug: I guess there was something suspect in it.

**Pull request merged**